other/appfilter.xml

```diff
@@ -1,4 +1,37 @@
 <resources>
+	<!-- Blacklist Blocker -->
+	<item component="ComponentInfo{com.kaliturin.blacklist/com.kaliturin.blacklist.activities.MainActivity}" drawable="blacklistblocker" />
+	
+	<!-- Calctastic -->
+	<item component="com.shaytasticsoftware.calctasticfree/com.calctastic.android.CalcTasticActivity}" drawable="calctastic" />
+	
+	<!-- gGompris -->
+	<item component="ComponentInfo{net.gcompris/net.gcompris.GComprisActivity}" drawable="gcompris" />
+
+	<!-- Natural Hour -->
+	<item component="ComponentInfo{com.forrestguice.suntimes.naturalhour/com.forrestguice.suntimes.naturalhour.MainActivity}" drawable="naturalhour" />
+
+	<!-- Open Flashlight -->
+	<item component="ComponentInfo{io.github.sanbeg.flashlight/io.github.sanbeg.flashlight.FlashLightActivity}" drawable="flashlight" />
+
+	<!-- Rxdroid -->
+	<item component="ComponentInfo{at.jclehner.rxdroid/at.jclehner.rxdroid.DrugListActivity2}" drawable="rxdroid" />
+
+	<!-- Solar Compass -->
+	<item component="ComponentInfo{com.agnibho.android.solarcompass/com.agnibho.android.solarcompass.MainActivity}" drawable="solarcompass" />
+
+	<!-- Solunar Periods -->
+	<item component="ComponentInfo{com.forrestguice.suntimes.solunar/com.forrestguice.suntimes.solunar.MainActivit}" drawable="solunarperiods" />
+
+	<!-- Tides -->
+	<item component="ComponentInfo{com.SeventhGear.tides.TidesApplication/com.SeventhGear.tides.MainActivity}" drawable="tides" />
+	
+	<!-- Sugarizer -->
+	<item component="ComponentInfo{org.olpc_france.sugarizer/org.olpc_france.sugarizer.MainActivity}" drawable="sugarizer" />
+
+	<!-- vlc -->
```

Comment:
I think this needs to be Speakingpal or something?

Reply:
Please tell me if you got the updated appfilter.xml from the 11 icons I commited 2 days ago. Now I was trying to merge my two branches but maybe I commited to yours... I'm getting a little confused

Reply:
No worries, github can be super confusing. I'll take a look.